docs/zh_CN/ch1/component_upgrade_cn.md

```diff
@@ -14,7 +14,7 @@ Exchangis 升级文档
 ```
 
 #### 2）下载二进制包
-我们提供ExchangisAppconn的升级物料包，您可直接下载使用。[点击跳转 Release 界面](https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/WeDatasphere/Exchangis/exchangis1.0.0-rc/exchangis-appconn.zip)
+我们提供ExchangisAppconn的升级物料包，您可直接下载使用。[点击跳转 Release 界面](https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/WeDatasphere/Exchangis/exchangis1.1.2/Exchangis1.1.2_install_package.zip)（TODO：待发版时更新）
```

Comment:
please delete 'TODO'

Reply:
It has been changed to the relative path.